web-ui/src/pages/FeedbackRequestPage.jsx

```diff
@@ -77,26 +81,35 @@ const FeedbackRequestPage = () => {
 
   const activeStep = getStep();
 
+  const hasFor = useCallback(() => {
+    return !!id;
+  }, [id])
+
   const hasTemplate = useCallback(() => {
     return !!templateQuery;
   }, [templateQuery])
 
-  const hasFor = useCallback(() => {
-    return !!forQuery;
-  }, [forQuery])
-
   const hasFrom = useCallback(() => {
     return !!fromQuery;
   }, [fromQuery])
-
-  const isValidDate = useCallback((dateString) => {
-    const timestamp = Date.parse(dateString);
-    return !isNaN(timestamp);
-  }, []);
-
-  const hasDue = useCallback(() => {
-    return (dueQuery && isValidDate(dueQuery))
-  }, [dueQuery, isValidDate]);
+  
+    const isValidDate = useCallback((dateString) => {
+      let today = new Date();
+      today = dateUtils.format(today, "yyyy-MM-dd");
+      let timeStamp = Date.parse(dateString)
+      if(dateString < today)
+        return false;
+      else
+        return !isNaN(timeStamp);
+     }, []);
+
+    const hasSend = useCallback(() => {
+      let isValidPair = false
+      if(dueQuery && dueQuery !== undefined) {
+        isValidPair = dueQuery >= sendQuery
+      }
+      return (sendQuery && isValidDate(sendQuery) && isValidPair)
```

Comment:
Would it make more sense to move the logic for `isValidPair` into the `isValidDate` function?